src/tools/spin/spinquic.cpp

```diff
@@ -268,6 +269,67 @@ QUIC_STATUS QUIC_API SpinQuicServerHandleListenerEvent(HQUIC /* Listener */, voi
     return QUIC_STATUS_SUCCESS;
 }
 
+void SpinQuicSetRandomSesssioParam(HQUIC Session)
+{
+    union {
+        uint64_t u64;
+        uint32_t u32;
+        uint16_t u16;
+        uint8_t  u8;
+        const char *cp;
+    } Param;
+    Param.cp = 0;
```

Comment:
If the goal is to zero out the union structure, you want to zero the u64 member, because on 32-bit systems, the pointer type will be smaller.